### DISCUSSION TITLE
Build LLVM preset on linux.

### DISCUSSION THREAD

**Discussion by Berstanio** [bytedeco/javacpp-presets#1073] [Q&A] [answered]
I try to build the llvm preset from source. I run "mvn clean package -Djavacpp.platform=linux-x86_64" in the root directory but the "llvm-linux-x86_64.jar" is empty (only contains the "META-INF" folder). If I run the same command in the "platform" directory I get the same result. What are the correct commands to built the platform specific jars? 
From the docs and the github actions I couldn't figure out what I do wrong.

Thanks in advance!

**Comment by saudet** [comment, marked as answer]
That should work, yes. You might just be using some buggy version of the JDK and/or Maven.
Please try to upgrade everything to the latest versions.

**Reply by Berstanio** [reply]
I reinstalled maven and now it works. Maybe I had some strange configuration. Thanks for your help!
